index.html

```diff
@@ -0,0 +1,87 @@
+                            fill="black" />
+                    </svg>
+                    <svg width="18" height="8" viewBox="0 0 18 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <rect y="0.333252" width="18" height="7.33333" rx="1.33333" fill="black" />
+                    </svg>
+                </div>
+                <div class="bottom-menu">
+                    <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                            d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z"
+                            fill="#6070FF" />
+                    </svg>
+                </div>
+            </div>
+        </div>
+        <div class="main-content">
+            <div class="container">
+                <div class="title">
+                    <p>Am Alphy glad to see you!</p>
+                </div>
+                <div class="description">
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some
+                        of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate to contact me.</p>
+                </div>
+                <div class="connect">
+                    <p>let's connect</p>
+                </div>
```

Comment:
- [ ] Kindly change this `<p>let's connect</p>` to `<a href="#">let's connect</a>` or to a button.  Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href. 
- [ ]  Remember to add the states as below
_**Button or Link states**_
![btn-1](https://user-images.githubusercontent.com/99656687/213596801-75ef0920-b7f6-4b7f-85b9-2e87a5879e41.png)